packages/wonder-blocks-data/docs.md

```diff
@@ -59,7 +74,7 @@ shown below.
 ```js static
 const responseCache = {
     HANDLER_TYPE_A: {
-        DATA_ID_1: {error: "It go 💥boom 😢"},
+        DATA_ID_1: {error: new Error("It go 💥boom 😢")},
```

Comment:
Darn it, accidentally undid this one too!

Reply:
OK, fixed it here: https://github.com/Khan/wonder-blocks/commit/8fc15f7964b96cccd35fc36c2a59f4f17dc72863